doc/source/whats_new/v0.13.2.txt

```diff
@@ -0,0 +1,9 @@
+.. currentmodule:: databroker
+
+v0.12.3 (2019-04-22)
+--------------------
+
+Bug Fixes
++++++++++
+
+* Support round trip of databroker configs, reporting the config, module and class
```

Comment:
Should we also add https://github.com/bluesky/databroker/pull/424 if it goes in and matters?

Reply:
Yes, but I figured in a separate PR (maybe in #424 itself?)

Reply:
Agreed via DM in Slack to skip mentioning the update from #424 as it does not affect the end users.